index.bs

```diff
@@ -410,34 +410,22 @@ to incoming responses. To do so, Fetch is patched as follows:
 
 ### Cross-Origin Resource Policy Checks ### {#corp-check}
 
-To perform a <dfn abstract-op>cross-origin resource policy check</dfn> given a [=request=]
-(|request|) and a [=response=] (|response|), run these steps:
-
-1.  Let |embedder policy| be "`require-corp`".
-
-2.  Set |embedder policy| to "`unsafe-none`" if both of the following statements are true:
-
-    *    |request|'s [=request/client=]'s [=environment settings object/embedder policy=]'s
-         [=embedder policy/value=] is "`unsafe-none`".
-    *    |request|'s [=request/reserved client=] is not `null`, and its
-         [=environment settings object/embedder policy=] is "`unsafe-none`".
-
-
-3.  Return `allowed` if any of the following statements are true:
+To perform a <dfn abstract-op>cross-origin resource policy internal check</dfn> given a string
+(|embedder policy value|), an origin (|origin|), a [=request=] (|request|) and a [=response=]
+(|response|), run these steps:
 
-    *    |request|'s [=request/mode=] is "`same-origin`", "`cors`", or "`websocket`".
-    *    |request|'s [=request/mode=] is "`navigate`", and |embedder policy| is "`unsafe-none`".
+1.  Return `allowed` if |request|'s [=request/mode=] is "`same-origin`", "`cors`", or "`websocket`".
 
-4.  ASSERT: |request|'s [=request/mode=] is "`no-cors`" or "`navigate`". If |request|'s
-    [=request/mode=] is "`navigate`", |embedder policy| is "`require-corp`".
+2.  If |request|'s mode is "`navigate`" and |embedder policy value| is "`unsafe-none`",
```

Comment:
Does this deal correctly with nested navigations via `<iframe>`/`<frame>`/`<embed>`/`<object>`? It's not clear to me that it does...

Reply:
This will be fixed in https://github.com/mikewest/corpp/pull/11. In short, we'll run the CORP check only for child browsing context navigation, so I removed the check here.

Reply:
Ok. Thanks. Perhaps we could turn this into an assertion that if the mode is `navigate`, the target is a nested browsing context?

Reply:
Done.